test_scripts/API/VehicleData/StabilityControlsStatus/012_SubscribeVehicleData_StabilityControlsStatus_and_other_parameters.lua

```diff
@@ -0,0 +1,65 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: Check subscription on StabilityControlsStatus and other parameters data
+--   via SubscribeVehicleData RPC
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) PTU is successfully performed
+-- 4) App is activated
+
+-- Steps:
+-- 1) App sends SubscribeVehicleData request with (stabilityControlsStatus = true, gps = true) to SDL
+-- SDL does:
+--  - send VehicleInfo.SubscribeVehicleData request with (stabilityControlsStatus = true, gps = true) to HMI
```

Comment:
@mvorobio sendS here and in all cases below

Reply:
@aderiabin What we have now:
```xml
APP sends ...
HMI sends...
SDL does:
- send...
```
If you don't want to specify every action in SDL - let's make it consistent with the HMI and apps. What we will have:

```xml
APP sends ...
HMI sends...
SDL sends...
```

Reply:
@aderiabin So at the end we will have: 
```
-- 1) App sends SubscribeVehicleData request with (stabilityControlsStatus = true, gps = true) to SDL
-- SDL sends VehicleInfo.SubscribeVehicleData request with (stabilityControlsStatus = true, gps = true) to HMI
-- HMI sends VehicleInfo.SubscribeVehicleData response "SUCCESS" with
--   (stabilityControlsStatus = { resultCode = "SUCCESS", dataType = "VEHICLEDATA_STABILITYCONTROLSSTATUS"},
--   gps = { resultCode = "SUCCESS", dataType = "VEHICLEDATA_GPS"})
-- SDL sends SubscribeVehicleData response with (success: true, resultCode: "SUCCESS") and received from HMI data to App
-- 2) HMI sends VehicleInfo.OnVehicleData notifications with StabilityControlsStatus and GPS data
-- SDL sends OnVehicleData notification with received from HMI data to App
```

Reply:
@AByzhynar I agree with proposed by you approach.

Reply:
@mvorobio `SDL does` can be removed in this case

Reply:
@mvorobio even if SDL will have to perform multiple actions, the following approach looks better due to : 
- Every line is context-independent
- Every line is self-descriptive
- Fewer lines in the code

```
APP sends ...
SDL updates...
SDL writes...
SDL sends...
HMI sends...
```

Reply:
@LitvinenkoIra, @AByzhynar  please check https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/15/commits/fce5bbcee34d46c939ce7f7c8801cdf53a572564